CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased Changes
 
+### FEATURES
+- [modules]
+  - Implementation the channel message handlers ([#316])
```

Comment:
Can you please add a link to the issue after line 14 below, otherwise the link won't work.

eg.

```
[#316]: https://github.com/informalsystems/ibc-rs/issues/316
```